core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java

```diff
@@ -43,6 +43,18 @@
  */
 public class WebUtils {
     
+    private static final String ENCODING_KEY = "encoding";
+    
+    private static final String ACCEPT_CHARSET_KEY = "Accept-Charset";
+    
+    private static final String COMMA = ",";
+    
+    private static final String SEMI = ";";
+    
+    private static final String JSON_AND_UTF8_CONTENT_TYPE = "application/json;charset=UTF-8";
```

Comment:
Can be replaced with
`com.alibaba.nacos.common.http.param.MediaType#APPLICATION_JSON`

Reply:
done